src/web/lib/components/ThemeCustomBackgroundPicker/index.js

```diff
@@ -129,12 +139,7 @@ export class ThemeCustomBackgroundPicker extends React.Component {
                   <React.Fragment>
                     <Modal>
                       <ul className="errors">
-                        {errors.tooLarge && (
-                          <li>The image is too large. (1MB maximum size)</li>
-                        )}
-                        {errors.wrongType && (
-                          <li>The file is not an accepted image type.</li>
-                        )}
+                        <li>{mapErrors(errors)}</li>
```

Comment:
Previously, more than one error message could be shown.

With this change, only one message is shown.

Reply:
hmm i see..I'll revisit. thanks!